Sources/StreamChatUI/ChatMessageList/Attachments/ChatMessageLinkPreviewView.swift

```diff
@@ -84,41 +84,24 @@ open class _ChatMessageLinkPreviewView<ExtraData: ExtraDataTypes>: _Control, The
         addSubview(authorBackground)
         addSubview(textStack)
         
-        var constraints: [NSLayoutConstraint] = []
-
-        imagePreview.pin(anchors: [.leading, .top, .trailing], to: layoutMarginsGuide)
-        constraints.append(
-            imagePreview.widthAnchor.pin(equalTo: imagePreview.heightAnchor)
-        )
-
-        textStack.addArrangedSubviews([
-            authorLabel,
-            titleLabel,
-            bodyTextView
-        ])
+        imagePreview.pin(anchors: [.leading, .top, .trailing], to: self)
+        imagePreview.widthAnchor.pin(equalTo: imagePreview.heightAnchor, multiplier: 2).isActive = true
+        
+        textStack.addArrangedSubviews([titleLabel, bodyTextView])
         textStack.axis = .vertical
         textStack.alignment = .leading
         textStack.spacing = 3
+        textStack.topAnchor.pin(equalToSystemSpacingBelow: imagePreview.bottomAnchor).isActive = true
         textStack.pin(anchors: [.leading, .bottom, .trailing], to: layoutMarginsGuide)
-        
-        bodyTextView.setContentHuggingPriority(.streamLow, for: .horizontal)
 
-        constraints += [
-            authorBackground.leadingAnchor.pin(equalTo: layoutMarginsGuide.leadingAnchor),
-            authorBackground.bottomAnchor.pin(equalTo: authorLabel.bottomAnchor),
-            authorBackground.layoutMarginsGuide.topAnchor.pin(equalTo: authorLabel.topAnchor),
-            authorBackground.layoutMarginsGuide.trailingAnchor.pin(equalTo: authorLabel.trailingAnchor)
-        ]
-
-        titleLabel.setContentCompressionResistancePriority(.streamRequire, for: .vertical)
+        authorBackground.leadingAnchor.pin(equalTo: imagePreview.leadingAnchor).isActive = true
+        authorBackground.bottomAnchor.pin(equalTo: imagePreview.bottomAnchor).isActive = true
+        imagePreview.trailingAnchor.pin(greaterThanOrEqualToSystemSpacingAfter: authorBackground.trailingAnchor).isActive = true
+        authorBackground.embed(authorLabel, insets: NSDirectionalEdgeInsets(top: 8, leading: 8, bottom: 4, trailing: 12))
```

Comment:
This feels strange to me: `NSDirectionalEdgeInsets(top: 8, leading: 8, bottom: 4, trailing: 12))`... Maybe add comment why is it like that? maybe trailing has something to do with some other insets?

Reply:
The values are from figma :)

Reply:
![Snímek obrazovky 2021-07-01 v 10 16 22](https://user-images.githubusercontent.com/7560049/124090759-6f574e00-da55-11eb-9609-c5e3bb080341.png)